test/controllers/responsiblepeople/VATRegisteredControllerSpec.scala

```diff
@@ -170,7 +171,9 @@ class VATRegisteredControllerSpec extends AmlsSpec with MockitoSugar with ScalaF
       }
     }
     "App" in {
-      val app = GuiceApplicationBuilder().build()
+      val app = GuiceApplicationBuilder()
```

Comment:
^

Reply:
same as above

Reply:
anyway I'll check some specs again

Reply:
Okay mate, just let me know once thats done and the rest looks good. Just seems pointless to me adding additional unit tests just to see whether or not the app runs. That should just be a given. I can see why its beneficial when amending the connectors to save you having to constantly check whether the page loads.

Reply:
you know tests above are not mine, looks like they are yours haha :P I was just updating some failing ones, seems I've added some unnecessary bindings, but I'm fixing that

Reply:
done